lib/services/role.go

```diff
@@ -1604,6 +1653,22 @@ func (m *AzureIdentityMatcher) String() string {
 	return fmt.Sprintf("AzureIdentityMatcher(Identity=%v)", m.Identity)
 }
 
+// GCPServiceAccountMatcher matches a role against GCP service account.
+type GCPServiceAccountMatcher struct {
+	ServiceAccount string
```

Comment:
nit: could we add a godoc comment and include an example ServiceAccount string?

Reply:
Will do.